### PR TITLE
Test move prover

### PR DESCRIPTION
**Pull request by junkil-park**
## Motivation

(Write your motivation for proposed changes here.)

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/libra/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

## Test Plan

(Share your test plan here. If you changed code, please provide us with clear instructions for verifying that your changes work.)

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/libra/website, and link to your PR here.)

### CODE REVIEW

File: .github/workflows/test-move-prover.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+    steps:
+    - name: Install Z3
+      uses: pavpanchekha/setup-z3@v1.2
+
+    - name: Install .NET Core SDK
+      run: |
+        sudo apt-get update
+        sudo apt-get install apt-transport-https
+        sudo apt-get update
```
Comment (sausagee) [inline .github/workflows/test-move-prover.yml]: duplicate?

File: .github/workflows/test-move-prover.yml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+    # - name : Install Boogie
+    #   run: |
+    #     sudo apt-get update
+    #     sudo apt-get install boogie=2.4.1+dfsg-0.1
+    #     sudo apt-get install boogie
+
+    # - name: Run Boogie
+    #   run: |
+    #     source ~/.bashrc
```
Comment (sausagee) [inline .github/workflows/test-move-prover.yml]: I don't see where the bash src is modified in this file. I supposed you would need to add boogie and/or z3 to the $PATH.

File: .github/workflows/test-move-prover.yml (hunk 1 above)
Comment (junkil-park) [reply .github/workflows/test-move-prover.yml]: Are you referring "sudo apt-get update"? This duplicate exists even in the official instruction for the .net core sdk installation. So, I kept it.

**Comment by junkil-park** [comment]
```test
commenting
test
```

**Pull request closed without merging**